projects/scullyio/ng-lib/src/lib/transfer-state/transfer-state.service.ts

```diff
@@ -80,7 +81,9 @@ export class TransferStateService {
     } else if (isScullyGenerated()) {
       // On the client AFTER scully rendered it
       this.initialUrl = window.location.pathname || '__no_NO_no__';
-      this.initialUrl = this.initialUrl.endsWith('/') ? this.initialUrl.slice(0, -1) : this.initialUrl;
+      this.initialUrl = /^\/?(?:[a-zA-Z0-9]+\/)+$/.test(this.initialUrl)
```

Comment:
Now that I think about it this regexp is not good enough because it will not crop trailing slash from urls like '/my_awesome-page/'
@SanderElias what should be supported characters in the routes?

Reply:
My other idea was to check much simpler like so:
this.initialUrl !== '/' && this.initialUrl.endsWith('/')`, maybe that would be still better?

Reply:
Yes, That would take care of the problem.

Reply:
I agree. I will change it to explicit check then.